src/qibocal/protocols/characterization/allxy/allxy.py

```diff
@@ -134,13 +134,17 @@ def add_gate_pair_pulses_to_sequence(
     gates,
     qubit,
     sequence,
+    sequence_delay=0,
+    readout_delay=0,
     beta_param=None,
 ):
     pulse_duration = platform.create_RX_pulse(qubit, start=0).duration
     # All gates have equal pulse duration
 
-    sequenceDuration = 0
-    pulse_start = 0
+    sequenceDuration = sequence.duration + sequence_delay
```

Comment:
I know this is a leftover from the past, but we shoud probably convert to the proper naming convention at some point
```suggestion
    sequence_duration = sequence.duration + sequence_delay
```